nfs_v3_config_test.go

```diff
@@ -0,0 +1,274 @@
+package nfsv3driver_test
+
+import (
+	"code.cloudfoundry.org/lager"
+	"code.cloudfoundry.org/lager/lagertest"
+	"code.cloudfoundry.org/nfsv3driver"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("BrokerConfigDetails", func() {
+	var (
+		logger lager.Logger
+	)
+
+	BeforeEach(func() {
+		logger = lagertest.NewTestLogger("test-broker-config")
+	})
+
+	Context("ConfigDetails", func() {
+		It("returns empty mount & source options, no mandatory", func() {
```

Comment:
please consider "given no mandatory and empty config, returns empty mount & source options, "